### PR TITLE
Fix quality

### PR DESCRIPTION
**Pull request by tkkuehn**
## Proposed changes

Get all our quality checkers passing.

I couldn't get pylint to work properly in pre-commit, but it passes on my machine too.

## Types of changes

What types of changes does your code introduce? Put an `x` in the boxes that apply

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionalitiy)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other (if none of the other choices apply)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you are unsure about any of the choices, don't hesitate to ask!

- [ ] Changes have been tested to ensure that fix is effective or that a feature works.
- [ ] Changes pass the unit tests
- [ ] Code has been run through [`black`](https://github.com/psf/black) with the `-l 79` flag.
- [ ] I have included necessary documentation or comments (as necessary)
- [ ] Any dependent changes have been merged and published

## Notes
All PRs will undergo the unit testing before being reviewed. You may be requested to explain or make additional changes before the PR is accepted.

_PR template was adopted from [appium](https://github.com/appium/appium/blob/master/.github/PULL_REQUEST_TEMPLATE.md)_

### CODE REVIEW

**Comment by kaitj** [comment]
Will take a closer look later this morning, but I think it's failing because of the two errors being thrown in the `pylint` on actions. We could either: 
1. Have it always return non-zero exit code (I don't love this idea)
2. Or add an ignore for those errors for those classes / functions (I think I would lean towards this).

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -16,17 +17,7 @@ repos:
     hooks:
     -   id: black
 
-# -   repo: https://github.com/pycqa/flake8
-#     rev: 5.0.4
-#     hooks:
-#     -   id: flake8
-
-# Pylint needs to be installed locally for precommit
```
Comment (kaitj) [inline .pre-commit-config.yaml:24]: Not opposed to not including the pylint in the pre-commit but just wondering if there was any particular reason for removing it?

File: test/test_model.py [hunk 2]
```diff
@@ -13,20 +16,22 @@ def test_empty_afids(self):
 
     def test_valid_afids(self):
         test_afids = model.HumanFiducialSet()
-        for label, descs in model.EXPECTED_MAP.items():
+        for _, descs in model.EXPECTED_MAP.items():
```
Comment (kaitj) [inline test/test_model.py:19]: This is minor, but we do we need to use `items()` here? Could we just return `values()` instead if we aren't using the label? 

Also just realized this is for testing, so could just ignore this comment all together.

**Review by kaitj — CHANGES_REQUESTED** [review]
Looks good overall. Few minor things:

1. Pylint precommit (see comment)
2. We probably don't want the .json and .fcsv files to be linted, which it looks like they were in this case.

File: .pre-commit-config.yaml (hunk 1 above)
Comment (tkkuehn) [reply .pre-commit-config.yaml:24]: I couldn't get it to work properly -- precommit didn't seem to play nicely with my local installation of pylint. It wouldn't pick up the `pylint_flask_sqlalchemy` plugin and ignored the `# pylint: disable=...` directives I added, leading it to fail every time (similar to what happened in the GH action).

**Comment by tkkuehn** [comment]
> Will take a closer look later this morning, but I think it's failing because of the two errors being thrown in the `pylint` on actions.

I think the `pylint_flask_sqlalchemy` dependency I added should fix this (at least it does on my machine). However, it clearly doesn't on the GH action, either because it's malfunctioning or the config in the `pyproject.toml` isn't being used for pylint. In any case, just adding the ignores inline is probably easiest.

**Comment by kaitj** [comment]
> > Will take a closer look later this morning, but I think it's failing because of the two errors being thrown in the `pylint` on actions.
> 
> I think the `pylint_flask_sqlalchemy` dependency I added should fix this (at least it does on my machine). However, it clearly doesn't on the GH action, either because it's malfunctioning or the config in the `pyproject.toml` isn't being used for pylint. In any case, just adding the ignores inline is probably easiest.

Hmm okay, I'll play around with it a little more this morning and see if I can get that part working then.

**Pull request merged**